rental.rb

```diff
@@ -0,0 +1,15 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Let's mark `book` and `person` with `attr_accessor`, as we intend to both _access (get)_ as well as _write (set)_ to it. Kindly ensure to improve upon this aspect.

Reply:
@rloterh  kindly confirm what you are telling me to change is it right?

Reply:
@Tracy-miranja 
Yep, let change the encapsulation of _book_ and _person_ to attr_accessor, like below:
```ruby
class Rental
  attr_accessor :date, :book, :person